# Generated by Django 2.1.1 on 2018-09-27 23:00

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('pronosticos', '0003_auto_20180927_1613'),
    ]

    operations = [
        migrations.CreateModel(
            name='Fecha',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(max_length=25)),
            ],
        ),
        migrations.CreateModel(
            name='Pronostico',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('local_gol', models.IntegerField(default=0)),
                ('visita_gol', models.IntegerField(default=0)),
                ('puntos', models.IntegerField(default=0)),
            ],
        ),
        migrations.AlterField(
            model_name='partido',
            name='fecha',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='fecha_partido', to='pronosticos.Fecha'),
        ),
        migrations.AddField(
            model_name='pronostico',
            name='partido',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='partido_pronostico', to='pronosticos.Partido'),
        ),
        migrations.AddField(
            model_name='pronostico',
            name='user',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='user_pronostico', to=settings.AUTH_USER_MODEL),
        ),
    ]
